tensornetwork/__init__.py

```diff
@@ -11,8 +11,6 @@
 from tensornetwork.visualization.graphviz import to_graphviz
 from tensornetwork import contractors
 from tensornetwork.utils import load_nodes, save_nodes
-from tensornetwork.matrixproductstates.finite_mps import FiniteMPS
-from tensornetwork.matrixproductstates.infinite_mps import InfiniteMPS
```

Comment:
Why are you removing these?

Reply:
pylint is detecting a cyclic import issue, and I have no idea where it comes from. Seems to be related to this import. I'm trying to track it down

Reply:
It's pretty weird: pylint reports
tensornetwork/contractors/opt_einsum_paths/path_contractors.py:1:0: R0401: Cyclic import (tensornetwork -> tensornetwork.matrixproductstates.finite_mps -> tensornetwork.matrixproductstates.base_mps) (cyclic-import)
tensornetwork/contractors/opt_einsum_paths/path_contractors.py:1:0: R0401: Cyclic import (tensornetwork -> tensornetwork.matrixproductstates.infinite_mps -> tensornetwork.matrixproductstates.base_mps) (cyclic-import)
No idea what this is about

Reply:
Do we even need to fill in anything in the `tensornetwork/matrixproductstates/__init__.py`? I'm confused why this file isn't empty. Ideally you should be able to access these classes just via `tn.FiniteMPS` and `tn.InfiniteMPS`

Reply:
had it cleared earlier but the linter kept complaining. it's super weird, everything was fine up to this PR. I'll try to find out what's going on

Reply:
I think this might be a bug in pylint.